test/specs/gfm/gfm.0.29.json

```diff
@@ -47,6 +47,18 @@
     "markdown": "| abc | def |\n| --- | --- |",
     "example": 205
   },
+  {
```

Comment:
You shouldn't change the `test/specs/gfm/ or `test/spec/commonmark/` json files as they are generated from the spec.

You can add these to the `test/specs/new/` folder. Just add a `.md` file with the markdown and a `.html` file with the same name with the expected html.

Reply:
Oh, I see.

Reply:
Changes to `test/specs/gfm` reverted. 
Test cases migrated to `test/specs/new/table_following_text.*`